src/Analyzers/CSharp/Tests/UseIndexOrRangeOperator/UseRangeOperatorTests.cs

```diff
@@ -330,25 +330,14 @@ class C
 {
     void Goo(S s)
     {
-        s.Slice([|1, s.Length - 2|]) = default;
+        s.Slice(1, s.Length - 2) = default;
```

Comment:
I would add [WorkItem(43202, "[https://github.com/dotnet/roslyn/issues/43202](https://github.com/dotnet/roslyn/issues/43202) ")] at the top so that we can find why this change was made